docs/source/learn/core_notebooks/pymc_aesara.ipynb

```diff
@@ -0,0 +1,1960 @@
+{
```

Comment:
<p>Nitpick, do you need the sum?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='91'/>

Reply:
I think so otherwise we get a vector right?
![image](https://user-images.githubusercontent.com/22996444/172025045-aa7f233e-a045-45ea-a2cf-29a4bd1d29f3.png)

Reply:
<p>My bad, I got confused with the multivariate normal comparison!</p>